models/yang/sonic/sonic-ifa.yang

```diff
@@ -0,0 +1,75 @@
+module sonic-ifa {
+    namespace "http://github.com/Azure/sonic-ifa";
+    prefix ifa;
+
+    import sonic-tam {
+        prefix stam;
+    }
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description
+        "SONIC TAM IFA";
+
+    revision 2019-05-15 {
+        description
+            "Initial revision.";
+    }
+
+    container sonic-ifa {
+
+        container TAM_INT_IFA_FEATURE_TABLE {
+
+            list TAM_INT_IFA_FEATURE_TABLE_LIST {
+                key "feature";
+                max-elements "1";
+
+                leaf feature {
+                    type string;
+                    default "feature";
+                }
+
+                leaf enable {
+                    type boolean;
+                }
+            }
+        }
+
+        container TAM_INT_IFA_FLOW_TABLE {
+
+            list TAM_INT_IFA_FLOW_TABLE_LIST {
+                key "name";
+
+                leaf name {
+                    type string;
+                }
+
+                leaf acl-table-name {
+                    type string;
+                }
+
+                leaf acl-rule-name {
+                    type string;
+                }
+
+                leaf sampling-rate {
+                    mandatory false;
```

Comment:
By default it is not mandatory. Please remove 'mandatory false'.

Reply:
Done